osf/management/commands/data_storage_usage.py

```diff
@@ -565,10 +613,28 @@ def process_usages(
     summary_data['nd_private_nodes'] = summary_totals.get('osf.private-node', 0)
     summary_data['nd_preprints'] = summary_totals.get('nd_preprints', 0)
     summary_data['nd_supp_nodes'] = summary_totals.get('nd_supplement', 0)
-    summary_data['canada_montreal'] = summary_totals.get(u'Canada - Montréal', 0)
-    summary_data['australia_sydney'] = summary_totals.get('Australia - Sydney', 0)
-    summary_data['germany_frankfurt'] = summary_totals.get('Germany - Frankfurt', 0)
-    summary_data['united_states'] = summary_totals.get('United States', 0)
+    summary_data['canada_montreal'] = summary_totals.get(CANADA, 0)
+    summary_data['australia_sydney'] = summary_totals.get(AUSTRALIA, 0)
+    summary_data['germany_frankfurt'] = summary_totals.get(GERMANY, 0)
+    summary_data['united_states'] = summary_totals.get(USA, 0)
+    # International Data
+    summary_data['num_nodes_usa'] = get_node_count_by_region(USA)
+    summary_data['num_quickfiles_usa'] = get_quickfile_count_by_region(USA)
```

Comment:
Looking through the requirements, all of the data for this ticket is supposed to go into one file, not split between these two. I think you're going to have to move all of these over to your new script, which might be more correctly called 'international_storage_metrics'. I you don't need to do raw data for the items that you have currently in here, because the raw data I'm capturing covers all of these things.

So: keep the raw data you currently are capturing, but move these summary items to the international_data_metrics script. Double check the requirements doc to make sure all that is correct.

Reply:
I remember discussing this with Courtney and she approved the whole two file thing back in 2019, I'm pretty sure it's still good.

Reply:
Ah, cool.